src/components/MyKiva/LendingStats.vue

```diff
@@ -366,6 +368,9 @@ export default {
 			this.userGoal = userGoal.value;
 			this.userGoalAchieved = userGoalAchieved.value;
 			this.goalProgressLoading = false;
+			this.storeGoalPreferencesFn = async updates => {
```

Comment:
Best way is calling the composable in setup hook, that's the way using a composable in options api...
I'll refactor this soon, no worries for now.

Reply:
Yeah thought about that.. Thanks!